### PR TITLE
Fix dev comments for string localization

### PR DESCRIPTION
**Pull request by electricRGB**

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1419,55 +1419,55 @@ extension String {
...
+    public static let SettingsCustomizeHomeStartAtHomeAfterFourHours = MZLocalizedString("Settings.Home.Option.StartAtHome.AfterFourHours", value: "Homepage after four hours of inactivity", comment: "In the settings menu, on the Start at Home homepage customization option, allows users to set this setting to return to the Homepage after four hours of inactivity.")
+    public static let SettingsCustomizeHomeStartAtHomeAlways = MZLocalizedString("Settings.Home.Option.StartAtHome.Always", value: "Homepage", comment: "In the settings menu, on the Start at Home homepage customization option, allows users to set this setting to return to the Homepage every time they open up Firefox")
+    public static let SettingsCustomizeHomeStartAtHomeNever = MZLocalizedString("Settings.Home.Option.StartAtHome.Never", value: "Last tab", comment: "In the settings menu, on the Start at Home homepage customization option, allows users to set this setting to return to the last tab they were on, every time they open up Firefox")
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1425]: To be a little more readable I'd add "this" before allows.

File: Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -1419,55 +1419,55 @@ extension String {
...
+    public static let SettingsCustomizeHomeStartAtHomeDescritpion = MZLocalizedString("Settings.Home.Option.StartAtHome.Description", value: "Choose what you see when you return to Firefox.", comment: "In the settings menu, in the Start at Home customization options, a description below the section describing what the section settings do.")
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1426]: nit: `SettingsCustomizeHomeStartAtHomeDescritpion` description spelling.

File: Client/Frontend/Strings.swift [hunk 3]
```diff
@@ -1419,55 +1419,55 @@ extension String {
...
-    public static let SettingsCustomizeHomeRecentSearches = MZLocalizedString("Settings.Home.Option.RecentSearches", value: "Recent Searches", comment: "Option in settings to toggle Recent Searches section on homepage")
-    public static let SettingsCustomizeHomeShortcuts = MZLocalizedString("Settings.Home.Option.Shortcuts", value: "Shortcuts", comment: "Option in settings to toggle Shortcuts section on homepage")
-    public static let SettingsCustomizeHomePocket = MZLocalizedString("Settings.Home.Option.Pocket", value: "Recommended by Pocket", comment: "Option in settings to turn on off pocket recommendations")
-    public static let SettingsCustomizeHomeDescritpion = MZLocalizedString("Settings.Home.Option.Description", value: "Choose content you see on the on Firefox homepage.", comment: "Description below the settings section describing what the section is for.")
+    public static let SettingsCustomizeHomeTitle = MZLocalizedString("Settings.Home.Option.Title", value: "Firefox Homepage", comment: "In the settings menu, the title of the Firefox Homepage customization settings section")
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1431]: nit: I'd add "this" before the title...

File: Client/Frontend/Strings.swift [hunk 4]
```diff
@@ -1419,55 +1419,55 @@ extension String {
...
+    public static let SettingsCustomizeHomeShortcuts = MZLocalizedString("Settings.Home.Option.Shortcuts", value: "Shortcuts", comment: "In the settings menu, in the Firefox homepage customization section, this option allows users to toggle Shortcuts section on the Firefox homepage on or off")
+    public static let SettingsCustomizeHomePocket = MZLocalizedString("Settings.Home.Option.Pocket", value: "Recommended by Pocket", comment: "In the settings menu, in the Firefox homepage customization section, this option allows users to turn the Pocket Recommendations section on the Firefox homepage on or off")
+    public static let SettingsCustomizeHomeDescritpion = MZLocalizedString("Settings.Home.Option.Description", value: "Choose content you see on the on Firefox homepage.", comment: "In the settings menu, on the Firefox homepage customization section, a description below the  section describing what the options in the section are for.")
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1438]: nit: "[this] is a description below..."

File: Client/Frontend/Strings.swift [hunk 5]
```diff
@@ -1419,55 +1419,55 @@ extension String {
...
-    public static let OnboardingWhatsNewTabGroupsDescription = MZLocalizedString("Onboarding.WhatsNew.TabGroups.Description", value: "Pages from the same search get grouped together.", comment: "Item Description")
-    public static let OnboardingWhatsNewRecentSearchesTitle = MZLocalizedString("Onboarding.WhatsNew.RecentSearches.Title", value: "Recent Searches", comment: "Title for an item on the card")
-    public static let OnboardingWhatsNewRecentSearchesDescription = MZLocalizedString("Onboarding.WhatsNew.RecentSearches.Description", value: "Revisit your latest searches from your homepage.", comment: "Item Description")
-    public static let OnboardingWhatsNewRecentButtonTitle = MZLocalizedString("Onboarding.WhatsNew.Button.Title", value: "Start Browsing", comment: "Title for the button used to get back to browsing on Firefox")
+    public static let OnboardingWhatsNewTitle = MZLocalizedString("Onboarding.WhatsNew.Title", value: "What’s New in Firefox", comment: "The title for the new onboarding card letting users know what in new in Firefox iOS")
```
Comment (nishant2718) [inline Client/Frontend/Strings.swift:1450]: letting users know what is* new

**Comment by nishant2718** [comment]
In general, I'd say it's good to add "this is" or something similar to be more specific. I haven't marked them all, but you get the idea.

**Review by nishant2718 — APPROVED** [review]
👌

**Comment by electricRGB** [comment]
Ignored BR because it' was just localization comment updates.

**Pull request merged**